Hyperbolic_surface_triangulation_2/doc/Hyperbolic_surface_triangulation_2/dependencies

```diff
@@ -0,0 +1,6 @@
+Manual
+Stream_support
+Number_types
+Combinatorial_map
+Hyperbolic_triangulation_2
+Algebraic_foundations
```

Comment:
Add `Periodic_4_hyperbolic_triangulation_2` to, probably, get the link in the documentation:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-6.1-Ic-65/doc/Hyperbolic_surface_triangulation_2/Hyperbolic_surface_triangulation_2.txt:20: warning: unable to resolve reference to 'Chapter_2D_Periodic_Hyperbolic_Triangulations' for \ref command
```
correct